patches/api/0348-Add-player-health-update-API.patch

```diff
@@ -0,0 +1,26 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SamB440 <sam@islandearth.net>
+Date: Wed, 17 Nov 2021 12:30:36 +0000
+Subject: [PATCH] Add player health update API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 9a6e410206852029f1fea0c4409352d5743dcf64..566b47da282ed0c8bc87b0f1757019f5c7e2acfb 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1558,6 +1558,15 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      */
+     public double getHealthScale();
+ 
++    // Paper start
++    /**
++     * Forcefully sends a health update to the player.
++     * This includes the current health, saturation, and food level.
++     * This may show the heart animation of gaining or losing health.
```

Comment:
In what situations will it/won't it show the animation? Might be worth documenting?

Reply:
I believe it will only show if there is a heart change, so afaik it will only work after using setHealth